scan-engine/src/main/java/de/tillhub/scanengine/helper/BundleExtension.kt

```diff
@@ -1,16 +1,10 @@
 package de.tillhub.scanengine.helper
 
-import android.content.Intent
 import android.os.Build
 import android.os.Bundle
 import java.io.Serializable
 
-inline fun <reified T : Serializable> Bundle.serializable(key: String): T? = when {
+internal inline fun <reified T : Serializable> Bundle.serializable(key: String): T? = when {
```

Comment:
Update code-ktx and use BundleCompact they support now getSerializable and you can delete then our BundleExtension